src/oatpp/web/server/handler/ErrorHandler.cpp

```diff
@@ -32,8 +32,10 @@ std::shared_ptr<protocol::http::outgoing::Response> ErrorHandler::handleError(co
 
   std::shared_ptr<protocol::http::outgoing::Response> response;
 
+#ifdef __GNUC__
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
@fhuberts , I'm not sure if we should hide deprecated methods usage - instead we want it to be fixed.

Reply:
Yeah I agree, but in one build it fails on the pragmas, and then in another build it also fails when I hid the pragmas.

So I don't know what to do.

Reply:
I've pulled out the commit "Wrap GCC pragma in '#ifdef __GNUC__'"